### PR TITLE
correct doc for Series.idxmax

### PR DESCRIPTION
**Pull request by Rubtsowa**

### CODE REVIEW

**Comment by AlexanderKalistratov** [comment]
> Please add the limitation to the docstring about unsupported `skipna`.

but `skipna` is supported...

**Comment by AlexanderKalistratov** [comment]
> > Please add the limitation to the docstring about unsupported `skipna`.
> 
> but `skipna` is supported...

No, it doesn't 😄

**Comment by AlexanderKalistratov** [comment]
> > > Please add the limitation to the docstring about unsupported `skipna`.
> > 
> > 
> > but `skipna` is supported...
> 
> No, it doesn't 😄

No, it doesn't for strings, but does for numeric data types

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 1]
```diff
@@ -2794,6 +2794,10 @@ def hpat_pandas_series_idxmax(self, axis=None, skipna=None):
 
     Pandas API: pandas.Series.idxmax
 
+    Limitations
+    -----------
+    Parameters axis, skipna are currently unsupported by Intel Scalable Dataframe Compiler.
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_series_functions.py]: `skipna` is supported restrictly. So reflect it here.

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 2]
```diff
@@ -2794,6 +2794,11 @@ def hpat_pandas_series_idxmax(self, axis=None, skipna=None):
 
     Pandas API: pandas.Series.idxmax
 
+    Limitations
+    -----------
+    Parameter axis is currently unsupported by Intel Scalable Dataframe Compiler.
+    Parameter skipna unsupported with str data.
```
Comment (AlexanderKalistratov) [inline sdc/datatypes/hpat_pandas_series_functions.py]: is unsupported

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 3]
```diff
@@ -2794,6 +2794,11 @@ def hpat_pandas_series_idxmax(self, axis=None, skipna=None):
 
     Pandas API: pandas.Series.idxmax
 
+    Limitations
+    -----------
+    Parameter axis is currently unsupported by Intel Scalable Dataframe Compiler.
+    Parameter skipna is unsupported with str data.
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_series_functions.py]:
```suggestion
    Parameter ``axis`` is supported only with default value ``None``.
    Parameter ``skipna`` cannot be ``False`` with data of string type.
```

**Pull request merged**